packages/nextjs/test/integration/pages/api/http/index.ts

```diff
@@ -9,4 +9,4 @@ const handler = async (_req: NextApiRequest, res: NextApiResponse): Promise<void
   res.status(200).json({});
 };
 
-export default withSentry(handler);
+export default wrapApiHandlerWithSentry(handler, '/api/http');
```

Comment:
shouldn't we be autowrapping these anyway?

Reply:
Actually yes. I removed the wrappers and everything still seems to pass 👍